packages/flutter/test/material/progress_indicator_test.dart

```diff
@@ -1077,13 +1077,39 @@ void main() {
       );
 
       expect(find.byType(CupertinoActivityIndicator), findsOneWidget);
-    },
-    variant: const TargetPlatformVariant(<TargetPlatform> {
+      },
+      variant: const TargetPlatformVariant(<TargetPlatform> {
+        TargetPlatform.iOS,
+        TargetPlatform.macOS,
+      }),
+    );
+
+  testWidgets(
+    'Adaptive CircularProgressIndicator displays CupertinoActivityIndicator in iOS',
+    (WidgetTester tester) async {
+      await tester.pumpWidget(
+        MaterialApp(
+          theme: ThemeData(),
+          home: const Scaffold(
+            body: Material(
+              child: CircularProgressIndicator.adaptive(
+                value:0.5,
+              ),
+            ),
+          ),
+        ),
+      );
+
+      expect(find.byType(CupertinoActivityIndicator), findsOneWidget);
+      final double actualProgress =tester.widget<CupertinoActivityIndicator>(
+        find.byType(CupertinoActivityIndicator)).progress;
+      expect(actualProgress, 0.5);
+      },
+      variant: const TargetPlatformVariant(<TargetPlatform> {
```

Comment:
```suggestion
      expect(actualProgress, 0.5);
   },
   variant: const TargetPlatformVariant(<TargetPlatform> {
```
Formatting nit, tab these two lines to the left